src/apple/system.rs

```diff
@@ -42,24 +47,30 @@ pub struct System {
     processors: Vec<Processor>,
     page_size_kb: u64,
     components: Vec<Component>,
+    // Used to get CPU information, not supported on iOS.
+    #[cfg(target_os = "macos")]
     connection: Option<ffi::io_connect_t>,
     disks: Vec<Disk>,
     networks: Networks,
     port: ffi::mach_port_t,
     users: Vec<User>,
     boot_time: u64,
     // Used to get disk information, to be more specific, it's needed by the
-    // DADiskCreateFromVolumePath function.
+    // DADiskCreateFromVolumePath function. Not supported on iOS.
+    #[cfg(target_os = "macos")]
     session: SessionWrap,
 }
 
 impl Drop for System {
     fn drop(&mut self) {
-        if let Some(conn) = self.connection {
+        #[cfg(target_os = "macos")]
+        if let Some(_conn) = self.connection {
```

Comment:
No need to rename `conn` anymore I think? :)

Reply:
Oh good catch, thank you.

Reply:
The rename has been reverted.